### PR TITLE
removeAccountsFromAList -> removeAccountsFromList

### PR DESCRIPTION
**Pull request by technicat**
consistent with addAccountsToList and RemoveAccountsFromListParams

https://github.com/TootSDK/TootSDK/discussions/138

### CODE REVIEW

File: Sources/TootSDK/TootClient/TootClient+Lists.swift [hunk 1]
```diff
@@ -96,7 +96,7 @@ public extension TootClient {
     }
     
     /// Add accounts to a list
```
Comment (davidgarywood) [inline Sources/TootSDK/TootClient/TootClient+Lists.swift:98]:
```suggestion
    /// Remove account from a list
```

**Review by davidgarywood — CHANGES_REQUESTED** [review]
A tiny nit on the documentation comment (I think that was our mistake not yours, but opportunity to clean up!)

**Review by davidgarywood — APPROVED** [review]
🙏

**Comment by kkostov** [comment]
I think it can be merged, if the linter is happy 😄

**Comment by technicat** [comment]
Swiftint is complaining about this line

TootClient+Account.swift:114:26: warning: Blanket Disable Command Violation: The disabled 'todo' rule should be re-enabled before the end of the file (blanket_disable_command)

**Comment by davidgarywood** [comment]
I'm going to approve this, and then fix the linter with a separate PR shortly

**Review by davidgarywood — APPROVED** [review]

**Pull request merged**